docs/development/l10n.rst

```diff
@@ -79,37 +79,37 @@ your native language:
 Translate a phrase
 ------------------
 
-Each string to be translated is shown in a **Source** box and you can
+Each translatable string is shown in a **Source** box and you can
 translate it right below in the **Translation** text area. When you
-are done, click **Save** and another string will be proposed.
+are done, click **Save** and the next untranslated string will appear.
 
 |Weblate translate screenshot|
 
-For most strings a screenshot is available on the right side to show
-how it looks in context. The screenshot can be displayed in full size
-by clicking on it:
+For most strings a contextual screenshot is available on the right side showing
+how it looks in the software. Click to see it in full size.
 
 |Weblate translate context screenshot|
 
 Instructions about a user navigation
 ------------------------------------
 
-The strings may contain instructions relative to the user interface of
-a software. The software should be run and the steps verified to find
-the translation. For instance::
+Strings also come with instructions, further explaining them, 
+and detailing where they are to be found in the user interface of the software, in case
+you still aren't sure. When doing so, also verify that the steps needed to find it
+are correct.::
 
   ...tap "Set up account"...
 
-must be translated by navigating to the localized interface of Google
-Authenticator and using the translation displayed in place of **Set up
+To find this, navigate to the localized interface of Google Authenticator to get a firmer grasp
```

Comment:
You can s/Google Authenticator/FreeOTP/ because it is now the recommended default.